app/javascript/src/components/Dashboard/Contacts/Table/TableColumn.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
```

Comment:
Why have you named this file as `TableColumn.jsx`?

Reply:
`TableColumn` replace with `getContactTableColumnData`

Reply:
@saurabh-nikam https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663318389500359?thread_ts=1663060378.971029&cid=C01T5Q4DZAR